client/src/components/Tool/ToolTutorialRecommendations.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+import Heading from "@/components/Common/Heading.vue";
+import { useToolTrainingMaterial } from "@/composables/toolTrainingMaterial";
+import ExternalLink from "../ExternalLink.vue";
+import { BCollapse, BButton } from "bootstrap-vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faCaretDown } from "@fortawesome/free-solid-svg-icons";
+import { useUid } from "@/composables/utils/uid";
+import slugify from "slugify";
+import { computed } from "vue";
+
+const props = defineProps<{
+    name: string;
+    id: string;
+    version: string;
+    repository?: string;
```

Comment:
repository shouldn't be a string, it's an object (in most other places that deal with tool shed repos) with `tool_shed, name, owner, changeset_revision` and maybe some other things. What you're passing in is the owner, I think that might be easier to read for others ?